sdk/storage/Azure.Storage.Files.Shares/src/ShareFileClient.cs

```diff
@@ -1770,6 +1815,9 @@ private async Task<Response<ShareFileCopyInfo>> StartCopyInternal(
                         SetArchiveAttribute = setArchiveAttribute
                     };
 
+                    modeCopyMode ??= ModeCopyMode.Source;
+                    ownerCopyMode ??= OwnerCopyMode.Source;
```

Comment:
TODO, if these are required for NFS but not allowed for SMB, we should remove these two lines.

Reply:
Fixed.